jdaviz/app.py

```diff
@@ -453,7 +453,7 @@ def get_data_from_viewer(self, viewer_reference, data_label=None,
 
         return data
 
-    def get_subsets_from_viewer(self, viewer_reference, data_label=None):
+    def get_subsets_from_viewer(self, viewer_reference, data_label=None, subset_type=None):
```

Comment:
New `subset_type` needs to be documented. Is "spatial" here only for spectroscopy context? What about imagers?

Reply:
"spatial" is specifically to get subsets made in image viewers. I just updated the dosctring to (hopefully) make that clearer.